src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -985,10 +985,10 @@
     short-summary: Enable cluster autoscaler.
   - name: --min-count
     type: int
-    short-summary: Minimum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specify the value in the range of [1, 1000]
```

Comment:
User could add a system mode nodepool, the value 0 would be invalid for this case.